docs/windows-agent-teams-bug-summary.html

```
@@ -0,0 +1,209 @@
+      <thead><tr><th>Location</th><th>Effect</th></tr></thead>
+      <tbody>
+        <tr><td>claude-agent-teams-shim-env.ts:38</td><td>forced in-process, skipped shim setup</td></tr>
+        <tr><td>cli/handlers/core.ts:62</td><td><code>orca claude-teams</code> threw <code>unsupported_platform</code></td></tr>
+        <tr><td>orca-runtime.ts:19278</td><td>forced in-process in the launch config</td></tr>
+        <tr><td>tui-agent-config.ts:62</td><td><strong>fourth guard, found later</strong> — detection skipped Windows, so Settings showed "Available to install" regardless</td></tr>
+      </tbody>
+    </table></div>
+
+    <h3>Cause 2 — the shim is invisible to Windows, and psmux steals it</h3>
+    <p>Windows <code>CreateProcess</code> appends <code>.exe</code> and <strong>never</strong> <code>.cmd</code>. Orca's Windows shim was <code>tmux.cmd</code>, so a bare <code>spawn("tmux")</code> could not see it and found whatever tmux port was installed:</p>
+    <div class="scroll"><pre><code>spawn("tmux", ["-V"], { shell: false })  ->  tmux 3.3.6   &lt;- psmux wins
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)